concordium-node/src/utils.rs

```diff
@@ -91,6 +131,8 @@ pub fn get_config_and_logging_setup() -> anyhow::Result<(config::Config, config:
             "enabled"
         }
     );
+
+    // FIXME: This is only the log level for consensus when log_config is used.
```

Comment:
Fixme?

Reply:
This is an issue introduced in another PR, so I simply added a note for now. It should be fixed in a separate PR and has nothing to do with the mac node :)